Cargo.toml

```diff
@@ -13,8 +13,9 @@ license = "MIT"
 gitlab = { repository = "whatisaphone/rlbot-rust" }
 
 [dependencies]
-libloading = "0.5.0"
-ratelimit = "0.4.3"
+libloading = "0.5"
```

Comment:
I believe libs should generally have looser dependency version requirements if possible. This allows a downstream user to potentially use the same version if they have the same dependency, thus reducing code bloat.

The lock file is meant for pinning exact versions, but again not recommended for libraries for the same reason: https://doc.rust-lang.org/cargo/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries

That said, given pre-1.0 versions, it probably makes sense to specify more exact versions, until `1.0` at least.

Reply:
I googled a little more and the patch versions are minimums, not exact.

https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html

So `0.4.3` means `^0.4.3` which means `>=0.4.3, <0.5.0`. Armed with this new knowledge, I think it's better to specify a minimum here unless we go out of our way to test with the older version. The only way you'd end up with dups would be if some other dependency specifies e.g. `>=0.4.0,<0.4.2` in which case they probably have a very good reason.

Reply:
I'll revert this, I had forgotten the specifics and also forgotten that most of the rust ecosystem is pre-1.0. At $dayjob, we use ruby with the `Gemfile` which has the same semantics as `Cargo.toml` (same authors), but since most everything is post-1.0, the rule of thumb is different.